test/acceptance/accessibility.py

```diff
@@ -22,42 +24,70 @@ def _check_axs(self):
                 "link-name",  # TODO: AC-196
             ]
         })
-        report = self.submission_page.a11y_audit.check_for_accessibility_errors()
+        page.a11y_audit.check_for_accessibility_errors()
 
 
-class SelfAssessmentAxsTest(OpenAssessmentAxsTest):
+class SelfAssessmentA11yTest(OpenAssessmentA11yTest):
     """
     Test the accessibility of the self-assessment flow.
     """
 
     def setUp(self):
-        super(SelfAssessmentAxsTest, self).setUp('self_only')
+        super(SelfAssessmentA11yTest, self).setUp('self_only')
 
-    def test_self_assessment_axs(self):
-        self._check_axs()
+    def test_self_assessment_a11y(self):
+        self.submission_page.visit()
+        self._check_a11y(self.submission_page)
 
 
-class PeerAssessmentAxsTest(OpenAssessmentAxsTest):
+class PeerAssessmentA11yTest(OpenAssessmentA11yTest):
     """
     Test the accessibility of the peer-assessment flow.
     """
 
     def setUp(self):
-        super(PeerAssessmentAxsTest, self).setUp('peer_only')
+        super(PeerAssessmentA11yTest, self).setUp('peer_only')
 
-    def test_peer_assessment_axs(self):
-        self._check_axs()
+    def test_peer_assessment_a11y(self):
+        self.submission_page.visit()
+        self._check_a11y(self.submission_page)
 
 
-class StudentTrainingAxsTest(OpenAssessmentAxsTest):
+class StudentTrainingA11yTest(OpenAssessmentA11yTest):
     """
     Test the accessibility of student training (the "learning to assess" step).
     """
     def setUp(self):
-        super(StudentTrainingAxsTest, self).setUp('student_training')
+        super(StudentTrainingA11yTest, self).setUp('student_training')
+
+    def test_student_training_a11y(self):
+        self.submission_page.visit()
+        self._check_a11y(self.submission_page)
+
 
-    def test_student_training_axs(self):
-        self._check_axs()
+class StaffAreaA11yTest(OpenAssessmentA11yTest):
+    """
+    Test the accessibility of the staff area.
+    """
+    def setUp(self):
+        super(StaffAreaA11yTest, self).setUp('peer_only', staff=True)
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    def test_staff_tools_panel_a11y(self):
```

Comment:
If you can also enable that area that is hidden in your sandbox, that would be terrific!